x/gamm/module.go

```diff
@@ -65,7 +65,9 @@ func (b AppModuleBasic) RegisterRESTRoutes(ctx client.Context, r *mux.Router) {
 }
 
 func (b AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *runtime.ServeMux) {
-	types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx))
+	if err := types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx)); err != nil {
+		return
+	}
```

Comment:
I'd prefer if we added a // ignore:errcheck, and got that working. Or at least leave err := types.RegisterQuery... on its own line, to maintain readability?

Reply:
Out of curiosity, why do you think err check for these places are unnecessary?